tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_008_pos.ksh

```diff
@@ -38,14 +38,27 @@
 #
 ################################################################################
 
+function test_s_run
+{
+	typeset snap=$1
+
+    log_must $ZFS destroy -d $snap
+    log_mustnot datasetexists $snap	
```

Comment:
Minor cstyle issue.  The originally test indented by 4 spaces but should have used tabs like the other test cases.  The same issue exists in several other of these tests.